lib/autorestPlugin/pluginHost.ts

```diff
@@ -59,30 +73,35 @@ extension.Add(
     await Promise.all(promises)
   })
 
+export interface Options extends specValidator.Options {
+  consoleLogLevel?: Unknown
+}
+
 export async function openApiValidationExample(
```

Comment:
>function [](start = 13, length = 8)

arrow functions ?  ```export const openApiValidationExample = ( ... ) => {}```

Reply:
same for other places

---
In reply to: [194472396](https://github.com/Azure/oav/pull/257#discussion_r194472396) [](ancestors = 194472396)